src/pages/CircleAdminPage/CircleAdminPage.tsx

```diff
@@ -323,13 +346,48 @@ export const CircleAdminPage = () => {
     }
   };
 
+  const getDecimals = ({ symbol }: { symbol: string | undefined }) => {
+    if (symbol) {
+      const v = findVault({ symbol });
+      if (v) return v.decimals;
+    }
+    return 0;
+  };
+
+  const findVault = ({
+    vaultId,
+    symbol,
+  }: {
+    vaultId?: number | undefined;
+    symbol?: string | undefined;
+  }) => {
+    return vaultsQuery?.data?.find(v =>
+      vaultId ? v.id === vaultId : v.symbol === symbol
+    );
+  };
+
+  const updateBalanceState = async (symbol: string): Promise<void> => {
```

Comment:
i've updated this in DistributionForm to take in the `vault_id` instead of `symbol` so it can handle multiple vaults of the same token & if there are tokens of the same symbol

Reply:
done